src/mlo_dir_conv.cpp

```diff
@@ -255,14 +239,7 @@ std::vector<miopen::solver::ConvSolution>
 FindImplicitGemmWrWAllSolutions(const miopen::ConvolutionContext& ctx,
                                 const miopen::AnyInvokeParams& invoke_ctx)
 {
-#if WORKAROUND_SWDEV_227826
-    if(miopen::IsEnabled(MIOPEN_DEBUG_IMPLICIT_GEMM_FIND_ALL_SOLUTIONS{}))
```

Comment:
If we want to reverse the default behavior, then let's simply change `IsEnabled()` to `!IsDisabled()` wherever `MIOPEN_DEBUG_IMPLICIT_GEMM_FIND_ALL_SOLUTIONS` is used. No other changes required for now, I think.

Reply:
fixed

Reply:
Let's minimize changes.

Reply:
Resolved by https://github.com/ROCmSoftwarePlatform/MIOpen/pull/488/commits/bbe1ffb1ac9cd4ad103405d4c0cd16d2e02f2596